### PR TITLE
Scafold React App

### PR DESCRIPTION
**Pull request by arunbhalli**
Pt : https://www.pivotaltracker.com/story/show/178002266
### User Story
```
As a dev team
To display the application to the user
We need a frontend client to display the data
```
### Proposed Changes
- scaffolds React app
- clean ups auto-generated comments and files
- changes App.jsx to function component

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (tochman) [inline src/App.css]: Please remove this file if you don't use it.

**Comment by tochman** [comment]
Please add link to PT story/chore

**Comment by LucasKnudsen** [comment]
Also PR description should link to designated PT chore/feature

File: src/App.css (hunk 1 above)
Comment (arunbhalli) [reply src/App.css]: Thank you and i removed it

**Pull request merged**